README.rst

```diff
@@ -25,6 +25,12 @@ The ``binary_root`` is a required configuration item, it defines where binaries
 live so that when a new binary is POSTed the service will use this path to save
 the binary to.
 
+distributions_root
```

Comment:
Because , /etc is owned by root and it's troublesome to have python write files there.

Also, having this configurable makes it really easy to test by setting this to a temp directory.